tfx_addons/feature_selection/requirements.txt

```diff
@@ -0,0 +1,4 @@
+scikit_learn==1.1.2
+tensorflow
+tfx
+tfx_bsl==1.9.0
```

Comment:
you can actually remove this from here as we should instead `pip install -e ".[feature_selection]"` (feel free to document that)